content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+============
+Event booths
+============
+
+The Odoo *Events* application provides users with the ability to create event booths, sell their
+availability, and manage their reservations.
+
+Configuration
+=============
+
+In order to create, sell, and manage booths for events, the *Booth Management* feature must be
+activated.
+
+To do that, navigate to :menuselection:`Events app --> Configuration --> Settings`, and tick the
+checkbox beside :guilabel:`Booth Management`. Then, click :guilabel:`Save`.
+
+.. image:: event_booths/booth-management-setting.png
+   :align: center
+   :alt: The Booth Management setting in the Odoo Events application.
+
+.. important::
+   When the :guilabel:`Booth Management` setting is activated, a new *Product Type* becomes
+   available on all product forms: *Event Booth*.
+
+   This is important because every created booth **must** be assigned a *Booth Category* on their
+   respective booth form, and every booth category **must** have an *Event Booth* product assigned
+   to it.
+
+Booth categories
+================
+
+With the *Booth Management* setting activated in the *Events* app, a *Booth Categories* option
+becomes available in the :guilabel:`Configuration` menu.
+
+To access the :guilabel:`Booth Category` dashboard, go to :menuselection:`Events app -->
+Configuration --> Booth Categories`. Doing so reveals a list of all the booth categories that have
+been created in the database.
+
+.. image:: event_booths/booth-category-page.png
+   :align: center
+   :alt: The Booth Category page in the Odoo Events application.
+
+On the :guilabel:`Booth Category` page, the following information for each booth category is listed:
+
+- :guilabel:`Name`: the name of the booth category.
+- :guilabel:`Create Sponsor`: if the checkbox in this column is ticked, when a booth with this
+  category is booked, a sponsor is created for the user who booked it.
+- :guilabel:`Product`: the *Event Booth* product associated with that specific booth category.
+- :guilabel:`Price`: the price of a booth in that booth category.
+
+When the :guilabel:`additional options` :icon:`oi-settings-adjust` icon, located to the far-right of
+the column titles, is clicked, a drop-down menu of additional column options appears. From the
+resulting drop-down menu, tick the checkbox beside :guilabel:`Sponsor Level` and/or
+:guilabel:`Sponsor Type` to reveal those columns on the :guilabel:`Booth Category` page.
```

Comment:
What do the Sponsor Level and Type fields do? is there another doc that elaborates on these?

Reply:
In this case, it simply reveals those specific columns. So, it would merely display the Sponsor Level and Sponsor Type on the Booth Categories page. While those are somewhat self-explanatory, I agree, that a sponsor-related doc would be nice. I'll add it to my to-do list, and knock it out at some point in the near future.